src/apipkg/__init__.py

```diff
@@ -191,18 +209,21 @@ def getmod():
             mod.append(x)
         return mod[0]
 
+    repr_result = _alias_mod_repr(modname, modpath, attrname)
```

Comment:
Probably inlining this will make things simpler.

Reply:
don